### PR TITLE
fixed issue where vulnerabilities were only getting logged when a fix…

### PR DESCRIPTION
**Pull request by yodahekinsew**
Fixing the issue where vulnerabilities were only getting logged when a fixable vulnerability was found. The current behavior after this fix is that the vulnerability check will fail if and only if a severe and fixable vulnerability is found; but the vulnerability check will always log all of the vulnerabilities it finds. Additionally, if a fix is not available for an issue, we now clear out some of the fields that would incorrectly imply to the user that there was a fix when logged.

### CODE REVIEW

File: cip.go [hunk 1]
```diff
@@ -286,6 +286,10 @@ func main() {
 
 		if *severityThresholdPtr >= 0 {
 			klog.Info("********** START (VULN CHECK) **********")
+			klog.Info("DISCLAIMER: Vulnerabilities are found as issues with " +
+				"package binaries within image dependencies, not necessarily " +
+				"with the image dependencies themselves. So a \"fixable\" " +
```
Comment (listx) [inline cip.go:291]: "For example, even though a fixed version of the binary is available, it does not necessarily mean that a new version of the image layer is available."

Comment (yodahekinsew) [reply cip.go:291]: Done

File: cip.go [hunk 2]
```diff
@@ -286,6 +286,10 @@ func main() {
 
 		if *severityThresholdPtr >= 0 {
 			klog.Info("********** START (VULN CHECK) **********")
+			klog.Info("DISCLAIMER: Vulnerabilities are found as issues with " +
+				"package binaries within image dependencies, not necessarily " +
```
[the remark this answers is not in the document]
Comment (yodahekinsew) [reply cip.go:290]: Done

**Comment by listx** [comment]
/lgtm
/approve

/hold

Remove hold when ready.

**Pull request merged**